setup.py

```diff
@@ -51,7 +51,8 @@ def read(fname):
     # 'botocore < 1.11.0'
     # Not used directly but allows boto GCE plugins to load.
     # https://github.com/GoogleCloudPlatform/compute-image-packages/issues/262
-    'google-compute-engine==2.8.12'
+    'google-compute-engine==2.8.12',
+    'paramiko==2.6.0',
```

Comment:
Why is the version pin necessary?

Reply:
If we include paramiko in setup.py, then this part becomes irrelevant:

https://github.com/RaRe-Technologies/smart_open/blob/1770807b0c8da2a34f749afa17ff70a715bc85de/smart_open/ssh.py#L43-L50

Could you please clean it up?

Reply:
paramiko is included only in test dependencies (and this is probably ok, we do not want to add such heavy package to main deps)

Reply:
> Why is the version pin necessary?

It isn't, removed.